frontend/src/v5/ui/components/dashboard/dashboardList/dashboardListItem/components/dashboardListItemTitle/dashboardListItemContainerTitle/dashboardListItemContainerTitle.component.tsx

```diff
@@ -65,13 +65,13 @@ export const DashboardListItemContainerTitle = ({
 			)}
 			selected={isSelected}
 			tooltipTitle={
-				hasRevisions ? (
+				container.hasStatsPending || hasRevisions ? (
 					<FormattedMessage id="containers.list.item.title.tooltip" defaultMessage="Launch latest revision" />
 				) : (
 					<FormattedMessage id="containers.list.item.title.tooltip.empty" defaultMessage="No revisions" />
 				)
 			}
-			disabled={!hasRevisions}
+			disabled={false}
```

Comment:
I don't this should be false. Previously this was disabled when a container was empty.
This Branch: ![image](https://github.com/3drepo/3drepo.io/assets/62481250/a95c1754-64f8-4a50-8d6a-6e7a75b73bfc)

Staging:  ![image](https://github.com/3drepo/3drepo.io/assets/62481250/d5b9a107-77cd-4379-b5b1-c62787439763)

Reply:
While loading you dont know if the container has revisions or not. I'll change it so that while loading is enabled but once loaded if the container has no revisions it will be disabled.